README.rst

```diff
@@ -2,6 +2,29 @@
 Resolwe Bioinformatics Python API
 =================================
 
+|build| |docs| |pypi_version| |pypi_pyversions| |pypi_downloads|
+
+.. |build| image:: https://travis-ci.org/genialis/resolwe-bio-py.svg?branch=master
+    :target: https://travis-ci.org/genialis/resolwe-bio-py
+    :alt: Build Status
+
+.. |docs| image:: https://readthedocs.org/projects/resolwe-bio-py/badge/?version=latest
```

Comment:
@dblenkus, please change `readthedocs.org` to `readthedocs.io` to follow [Read the Docs subdomains hosting change](http://blog.readthedocs.com/securing-subdomains/). We'll also have to update other projects eventually.